cg/apps/housekeeper/hk.py

```diff
@@ -164,7 +164,7 @@ def get_files_from_version(version: Version, tags: set[str]) -> list[File] | Non
     def get_file_from_version(version: Version, tags: set[str]) -> File | None:
         """Return the first file matching the given tags."""
         files: list[File] = HousekeeperAPI.get_files_from_version(version=version, tags=tags)
-        return files[0] if files else None
+        return next((file for file in files), None)
```

Comment:
What does the "(file for file in files)" do here? Turn the list into something more suitable for next()?

Reply:
Not much... I had to change it because SonarCloud was complaining: https://github.com/Clinical-Genomics/cg/runs/22621038550.

Suggestions are welcomed because I didn't quite get why since we were already implementing the suggested solution 😓

Reply:
huh, weird. can't really see a reason to complain but it is what it is.